### ISSUE TITLE
device claim failed error message

### ISSUE THREAD

**Issue by kennethlimcp** [spark/particle-cli#241]
If claiming is not successful with `particle device add xxxx` , you see:

```
Failed to claim device, server said: [object Object]
```

**Comment by monkbroc** [comment]
Bryce is fixing this right now.

**Comment by brycekahle** [comment]
What did you use for `xxxx`? I'm guessing an invalid device id or a device that is not online. I just tested the transfer case and it works properly.

**Comment by kennethlimcp** [comment]
I used a valid device id. The issue is with ` [object Object]` in the output message and not the function call itself.

**Comment by brycekahle** [comment]
Yeah, if you issue a raw HTTP request, there is no error message just `{}`. Is the device online?

**Comment by kennethlimcp** [comment]
Hmm....i got this from: http://community.particle.io/t/photon-setup-failure-new-led-flashing-sequence/22277/25

It seems to have to do with new factory photons that are unclaimed and offline..

**Comment by brycekahle** [comment]
Yes, you must be online to claim that way.

**Comment by kennethlimcp** [comment]
TLDR; it might be better to somehow indicate a better error since that doesn't really help with new users having issues claiming their devices and failed..

**Comment by brycekahle** [comment]
It is a tricky balance because we do not want to leak information about what device ids are valid. This would allow id enumeration.
